js/memory.js

```diff
@@ -1,9 +1,40 @@
+
 class MemoryGame {
-  constructor(cards){
+  constructor(cards, pickedCards,pairsClicked, pairsGuessed){
     this.cards = cards;
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
     // add the rest of the class properties here
```

Comment:
You can add a function call here so that your cards will get shuffled automatically from the beginning
```
this.shuffleCards(cards);